### DISCUSSION TITLE
Manage wine versions, list doesn't serve it's purpose anymore.

### DISCUSSION THREAD

**Discussion by JoneKone** [lutris/lutris#4447] [General]
Manage wine versions list is.. confusing and .. overall not serving it's purpose of managing.

For a person who has some difficulty reading a huge list (read wall of text) of text and maintaining he's, shall I say shorting.

It lacks the shorting order and even if it had one, that would be a temporary fix. The GUI needs to be reworked. It has become a wall of text instead of a list. Partly because naming isn't unified anymore partly because there are so many different options now.

I'm having difficulty and I know what I'm looking for. A new person doesn't even know what he is looking at, without searching for wiki or duck.it.. 

Many ppl might say well you don't have to use it so often so it's fine, but to that I say many ppl who use it for the first time could get confused and ... drop the whole program cause .. never mind I didn't want to touch this anyways. 

![Screenshot from 2022-08-26 15-18-47](https://user-images.githubusercontent.com/1114115/186902397-9cee57b4-7c73-4355-8eba-16298989460d.png)

**Comment by danieljohnson2** [comment]
Do you have any ideas about how to improve this?

I can think of one thing. The check-boxes on the left are not clickable, you have to use the buttons on the right. Perhaps we should replace them with a suitable icon (or nothing when the WINE version is not installed), to avoid giving a false affordance.

**Reply by JoneKone** [reply]
Search feature. Some kind of Description, shorting.... A pull down menu to select different type's also it need's to be put in settings.

**Reply by danieljohnson2** [reply]
Do you mean sorting?

The thing is, all we have are the names you see. It is worthwhile to sort on or search that?

A description could be useful, but where do we get one?

**Reply by JoneKone** [reply]
... well I mean "show only ge protons" or "lutris protons" or "wine" And I feel this "menu" should include a + sign that says add your own.. so maybe you can manage your own...  "Fshack" needs a description "GE" ... etc..
